### PR TITLE
Auto ocp-22734

### PR DESCRIPTION
**Pull request by rhpmali**
@xingxingxia @wangke19 please review. Thanks

### CODE REVIEW

**Comment by xingxingxia** [comment]
@rhpmali before review, pls indicate if it already passed local test or Jenkins test (if Jenkins test, pls give passed job; during the PR iterative review-update-review process, at least one time test and pass Jenkins job)?

**Comment by rhpmali** [comment]
@xingxingxia Local test and jenkins jobs are successfully executed. Here is the jenkins job link : https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3-smoke/1241/consoleFull

**Comment by xingxingxia** [comment]
@rhpmali lgtm. Intended to @ the mergers to review and merge, but please first remove the commented out not needed code (you can save somewhere else if you want)
Another point, like my and Ke Wang's upgrade case PRs show in comments, please hide any internal urls in this public repo.

**Comment by rhpmali** [comment]
Removed commented line and removed internal link from comment.

**Comment by xingxingxia** [comment]
/lgtm
@liangxia @akostadinov pls help review and merge, thanks!

File: features/upgrade/api_auth/upgrade.feature [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    # Check the payload info
+    Given evaluation of `"oc adm release info --registry-config=/var/lib/kubelet/config.json <%= cb.payload_image %>"` is stored in the :oc_adm_release_info clipboard
+    When I store the ready and schedulable masters in the clipboard
+    And I use the "<%= cb.nodes[0].name %>" node
+    
+    # due to sensitive, didn't choose to dump and save the config.json file
+    # for using the step `I run the :oadm_release_info admin command ...`
+    
+    And I run commands on the host:
+      | <%= cb.oc_adm_release_info %> --image-for=cluster-authentication-operator      |
```
Comment (liangxia) [inline features/upgrade/api_auth/upgrade.feature:48]: Extra white spaces between `operator` and `|`

Comment (xingxingxia) [reply features/upgrade/api_auth/upgrade.feature:48]: @rhpmali , yes, core auto members have stricter requirement in format, therefore in future PRs pay more attention to space, indentation, alignment issues like this and others https://github.com/openshift/cucushift/pull/7695#discussion_r423415046 / https://github.com/openshift/cucushift/pull/7695#discussion_r423415134 to make the merge quicker.

**Comment by rhpmali** [comment]
Fixed indention issue, please review

**Comment by xingxingxia** [comment]
@rhpmali remember to move the case to Automated.

**Pull request merged**